test/framework/cloudstack.go

```diff
@@ -410,7 +410,7 @@ func (c *CloudStack) ClusterStateValidations() []clusterf.StateValidation {
 
 // WithKubeVersionAndOS returns a cluster config filler that sets the cluster kube version and the right template for all
 // cloudstack machine configs.
-func (c *CloudStack) WithKubeVersionAndOS(kubeVersion anywherev1.KubernetesVersion, os OS, release *releasev1.EksARelease, _ ...bool) api.ClusterConfigFiller {
+func (c *CloudStack) WithKubeVersionAndOS(kubeVersion anywherev1.KubernetesVersion, os OS, release *releasev1.EksARelease, _ ...string) api.ClusterConfigFiller {
```

Comment:
```suggestion
func (c *CloudStack) WithKubeVersionAndOS(kubeVersion anywherev1.KubernetesVersion, os OS, release *releasev1.EksARelease) api.ClusterConfigFiller {
```

Reply:
Same as above - I can't remove the unused arg since I need to have the 4th argument of type `...string` in those methods to satisfy the provider interface.